### PR TITLE
Fix omnomcom wrapped

### PR DESCRIPTION
**Pull request by ysbrandB**
Make use of the new endpoints made in https://github.com/saproto/saproto/pull/2278.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -10,15 +10,9 @@ const steps = 5;
 const currentStep = ref(0);
```
Comment (JonaMata) [inline src/App.vue:10]: Don’t forget to correct this for the deleted steps

Comment (JonaMata) [reply src/App.vue:10]: Maybe find a new way to the progress bar if there’s only so little steps

**Review by JonaMata — CHANGES_REQUESTED** [review]
LAGTM

File: src/App.vue (hunk 1 above)
Comment (ysbrandB) [reply src/App.vue:10]: The data crunching still takes some time so I think the current implementation still makes some sense

**Review by JonaMata — APPROVED** [review]
LGTM

**Pull request merged**